README.md

```diff
@@ -29,8 +29,7 @@ You can try last beta build by accessing our [TestFlight Public Link](https://te
 If you have already everything installed, opening the project workspace in Xcode should be as easy as:
 
 ```
-$ xcodegen                  # Create the xcodeproj with all project source files
-$ pod install               # Create the xcworkspace with all project dependencies
+$ ./setup_project.sh        # Local script that creates the xcodeproj and xcworkspace with all source files and dependencies
```

Comment:
I would prefer to keep using the "xcodegen / pod install" combo here for simplicity and for easy understanding of what is happening.

I never needed to use more complex setup. You could mention setup_project.sh in [INSTALL.md](https://github.com/vector-im/element-ios/blob/develop/INSTALL.md) for advanced usage. While possible, I prefer to keep thing simple at first place.